data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+				`	"I think the event would have been more interesting had I known what exactly was going on."`
+			`	The elevator finishes its descent and the doors open. The Heliarch agent nods at your response. "Hope that you enjoy this next event, I do," she says as she begins guiding the audience along a path leading to the next stadium. Upon entering, you find a rectangular court of some sort in the center. Small mounds dot the stadium grounds, while six large holes are arranged evenly around the sides of the field. Tall nets stretching to the ceiling separate the field and the seating area.`
+			`	After the audience is seated, an announcer once again comes on the loudspeaker. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting, and the next event will be starting shortly. As you sit waiting among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games.`
+			`	Half an hour passes before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. The members of each team are introduced to the crowd and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
+			`	The game starts. It's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience, but the nets are sturdy, and prevent any accidents.`
+			`	Eventually, a sphere finds its way into a hole, which lights up with the color of the team who scored: red. Much of the crowd jumps up and screams. You're not sure if everyone is a fan of the red team or if they're cheering simply because a point was scored. The sphere is brought back up automatically via a tube close to the hole and it's back in the game.`
+			`	The game continues, and each team accumulates a few points. The blue team, leading, scores a fifth point, and the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The Kimek with the translator explains what is happening.`
+			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, going from the team that scored the least all the way to the blue team at the end. Between the two events, Ulmodi is placed third. He and the rest of the top twenty contestants celebrate by running laps around the arena.`
```

Comment:
"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi has risen to being placed third, and you notice that the Saryd who caught your eye in the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`